src/lib/jitsi.ts

```diff
@@ -0,0 +1,43 @@
+import { JwtHeader } from 'jsonwebtoken';
+import getEnv from '../config/configuration';
+
+export const getJitsiJWTSecret = () => {
+  return Buffer.from(getEnv().secrets.jitsiSecret, 'base64').toString('utf8');
+};
+
+export const getJitsiJWTPayload = (id: string, name: string, email: string) => {
+  const now = new Date();
+  const appId = getEnv().jitsi.appId;
+  return {
+    aud: 'jitsi',
+    context: {
+      user: {
+        id,
+        name,
+        avatar: '',
+        email,
+        moderator: 'true',
+      },
+      features: {
+        livestreaming: 'true',
+        recording: 'true',
```

Comment:
We do not want to enable recording nor transcription nor livestreaming, etc for now. Just enough to do videocalls, nothing else